packages/mui-styled-engine/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@mui/styled-engine",
-  "version": "5.15.14",
+  "version": "5.16.1",
```

Comment:
Same scenario as with `@mui/material-nextjs`. This is not the expected version but I used the stable public version. I think it makes sense if `@mui/styled-engine` is stable, but let me know if I'm wrong.

Reply:
Agreed